apple-nav/src/Products/IPhone.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+
+const IPhone = props => {
+    return (
+      <div className="subNavDiv">
+        <div className="productDiv">
+          {props.data.map(product => {
+            return (
+              <div key={product.name}>
```

Comment:
Good place to work on making the code drier. This code is repeated in several components and the logic can be incorporated into a `<Product product={product} />`. Product component is in charge of rendering the array of products - one product at a time.